nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -77,6 +82,9 @@ CollisionMonitor::on_activate(const rclcpp_lifecycle::State & /*state*/)
 
   // Activating lifecycle publisher
   cmd_vel_out_pub_->on_activate();
+  if (state_pub_ != nullptr) {
```

Comment:
```suggestion
  if (state_pub_) {
```

Is enough - its the same but cleaner. Same for deactivate.